src/TensorFlowNET.Keras/Engine/Model.Evaluate.cs

```diff
@@ -70,13 +70,19 @@ public partial class Model
             return evaluate(data_handler, callbacks, is_val, test_function);
         }
 
-        public Dictionary<string, float> evaluate(IEnumerable<Tensor> x, Tensor y, int verbose = 1, bool is_val = false)
+        public Dictionary<string, float> evaluate(
+            IEnumerable<Tensor> x, 
+            Tensor y, 
+            int verbose = 1,
+            NDArray sample_weight = null,
```

Comment:
It's recommended to put `sample_weight` the last parameter, to keep its backward compatibility. However if this is exactly the order in python, just keeping it is okay.

Reply:
Yes, it follow the order in python.